tests/manifests/import-vm-v2v-apb.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: v2v-template
+  annotations:
+    description: "Template for v2v import from VMWare"
+objects:
+- apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ServiceInstance
+  metadata:
+    name: import-vm-apb
+  spec:
+    clusterServiceClassExternalName: ${APB_NAME}
+    clusterServicePlanExternalName: vmware
```

Comment:
What about `vmware-template`? Do we want to verify it as well?

Reply:
yes, included